src/Template/Layout/default.ctp

```diff
@@ -137,6 +137,7 @@
     <?= $this->Html->script('vendor/what-input.js') ?>
     <?= $this->Html->script('vendor/foundation.js') ?>
     <?= $this->Html->script('app.js') ?>
+    <?= $this->Html->script('scripts.js') ?>
```

Comment:
ha a script általános mindenhol elérhető functiont tartalmaz akkor az menjen az app.js-be. Ha nem akkor pedig egy controllername.methdname.js file-ba és csak az adott view hívja be